pkg/devfile/versions/common/components.go

```diff
@@ -0,0 +1,23 @@
+package common
+
+import "fmt"
+
+// ValidateComponents validates all the devfile components
+func ValidateComponents(components []DevfileComponent) error {
+
+	// 1. Check wether component of type dockerimage is present
+	isDockerImageComponentPresent := false
+	for _, component := range components {
+		if component.Type == DevfileComponentsTypeDockerimage {
+			isDockerImageComponentPresent = true
```

Comment:
shouldn't we `break` here once we have found the  `DockerImageComponent`?

Reply:
Yes, we should. Thanks!

Reply:
Done